### PR TITLE
openfst: init at 1.6.3

### PR DESCRIPTION
**Pull request by dfordivam**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/openfst/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "openfst";
+  version = "1.6.3";
+
+  src =fetchurl {
```
Comment (fpletz) [inline pkgs/development/libraries/openfst/default.nix]: One extra space after the = please. :)

File: pkgs/development/libraries/openfst/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      Library for constructing, combining, optimizing, and searching weighted finite-state transducers (FSTs).
+      FSTs have key applications in speech recognition and synthesis, machine translation, optical character recognition,
+      pattern matching, string processing, machine learning, information extraction and retrieval among others
+    '';
+    homepage = http://www.openfst.org/twiki/bin/view/FST/WebHome;
+    license = stdenv.lib.licenses.asl20;
+    maintainers = [ stdenv.lib.maintainers.dfordivam ];
+    platforms = stdenv.lib.platforms.linux;
+  };
+}
```
Comment (fpletz) [inline pkgs/development/libraries/openfst/default.nix]: Please add a newline here.

**Review by fpletz — COMMENTED** [review]
Only a few small issues with my OCD… Looks good otherwise! :+1:

**Comment by dfordivam** [comment]
Done

File: pkgs/development/libraries/openfst/default.nix (hunk 2 above)
Comment (dfordivam) [inline pkgs/development/libraries/openfst/default.nix]: Just curious.. why new line here?

Comment (fpletz) [inline pkgs/development/libraries/openfst/default.nix]: It is just common practice at least in the Unix world to end files with a newline. There is of course some discussion but this isn't a holy war like tabs vs. spaces. :smile:

**Pull request merged**